### PR TITLE
Added correct license and updated author names

### PR DESCRIPTION
**Pull request by ianmkenney**
- Changed the license to be GPLv3 (#8)
- Tracked down author names from previous MDAnalysis commits to AUTHORS that weren't obvious from git blame

### CODE REVIEW

**Comment by ianmkenney** [comment]
@IAlibay the issue was just with the license file being v2 right?

**Review by orbeckst — APPROVED** [review]
ok

**Review by IAlibay — CHANGES_REQUESTED** [review]
sorry for the quick review - please update the relevant classifiers in setup & pyproject files?

**Review by IAlibay — CHANGES_REQUESTED** [review]
So by updating the license to GPLv3 we'll also have to update the headers to state that they are GPLv3+ such as: https://github.com/MDAnalysis/hole2-mdakit/blob/1908af6c3decdd8efd9ed41b3de83add45adac38/mdakithole2/analysis/hole.py#L8


Note: normally it's not an issue, but in this case because some of the _code_ itself is claimed to be GPLv3, then we can't allow for the backwards compatibility (it's weird and I dislike it, but I think that's the safest approach to ensure compatibility).

File: setup.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
     long_description_content_type="text/markdown",
     version=versioneer.get_version(),
     cmdclass=versioneer.get_cmdclass(),
-    license='GNU Public License v2+',
+    license='GNU Public License v3',
```
Comment (IAlibay) [inline setup.py]:
```suggestion
    license='GNU Public License v3+',
```
?

**Review by IAlibay — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (ianmkenney) [reply setup.py]: Done

**Review by IAlibay — APPROVED** [review]

**Pull request merged**